paasta_tools/cli/utils.py

```diff
@@ -738,10 +738,7 @@ def short_to_full_git_sha(short, refs):
     :param refs: A list of refs in the git repository
     :return: The full git sha or None if one can't be found
     """
-    for sha in list(set(refs.values())):
-        if sha.startswith(short):
-            return sha
-    return None
+    return [sha for sha in set(refs.values()) if sha.startswith(short)]
```

Comment:
I would rather have the error handling happen once here instead of by the caller.

There is no room for having ambiguous git shas.

Don't even "raise", just exit 1 and print the friendly error message. This is in cli utils, it should only be used by user-interacting cli tools.